rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -2400,14 +2400,27 @@ void VKGSRender::patch_transform_constants(rsx::context* ctx, u32 index, u32 cou
 		data_source = iobuf.data();
 	}
 
+	// Preserving an active renderpass across a transfer operation is illegal vulkan. However, splitting up the CB into thousands of renderpasses incurs an overhead.
+	// We cheat here for specific cases where we already know the driver can let us get away with this.
+	static const rsx::simple_array<vk::driver_vendor> s_allowed_vendors =
+	{
+		vk::driver_vendor::AMD,
+		vk::driver_vendor::RADV,
+		vk::driver_vendor::LAVAPIPE,
+		vk::driver_vendor::NVIDIA
```

Comment:
This might be a good occasion to add NVK detection and explicitly call it like we differentiate AMD and RADV

Reply:
Sure. If the ANV issue is fixed I can try and inspect what NVK looks like. Though it was really slow last time I tried it.